eng/vmr-msft-comparison-baseline.json

```diff
@@ -12,11 +12,60 @@
   {
     "issueType": "AssemblyVersionMismatch",
     "descriptionMatch": ".resources.dll",
+    "allowRevisionOnlyVariance": true,
     "justification": "Resource dlls get per-build versioning but this doesn't matter."
   },
   {
     "issueType": "MissingShipping",
     "descriptionMatch": "Aspire.*",
     "justification": "Aspire to be removed from the VMR. Also .NET does not ship these packages."
+  },
+  {
+    "issueType": "AssemblyVersionMismatch",
+    "descriptionMatch": "Microsoft\\.CodeAnalysis.*\\.dll",
+    "allowRevisionOnlyVariance": true,
+    "justification": "CodeAnalysis dlls vary build to build with patch number."
+  },
+  {
+    "issueType": "AssemblyVersionMismatch",
+    "descriptionMatch": "Microsoft\\.Build\\.CodeAnalysis.*\\.dll",
+    "allowRevisionOnlyVariance": true,
+    "justification": "CodeAnalysis dlls vary build to build with patch number."
+  },
+  {
+    "issueType": "AssemblyVersionMismatch",
+    "descriptionMatch": "Microsoft\\.Build.*\\.CodeAnalysis\\.dll",
+    "allowRevisionOnlyVariance": true,
+    "justification": "CodeAnalysis dlls vary build to build with patch number."
+  },
+  {
+    "issueType": "AssemblyVersionMismatch",
+    "descriptionMatch": "Metrics.*\\.exe",
+    "allowRevisionOnlyVariance": true,
+    "justification": "Analyzers vary build to build"
+  },
+  {
+    "issueType": "AssemblyVersionMismatch",
+    "descriptionMatch": "Microsoft\\.NET\\..*\\.Tasks\\.dll",
+    "allowRevisionOnlyVariance": true,
+    "justification": "Allow build-build patch variance"
+  },
+  {
+    "issueType": "AssemblyVersionMismatch",
+    "descriptionMatch": "NuGet\\..*\\.dll",
+    "allowRevisionOnlyVariance": true,
+    "justification": "Nuget DLLs vary build to build."
+  },
+  {
+    "issueType": "AssemblyVersionMismatch",
+    "descriptionMatch": "Microsoft\\.DotNet\\..*\\.dll",
+    "allowRevisionOnlyVariance": true,
+    "justification": "Arcade binaries vary build to build."
+  },
```

Comment:
Not sure about this one. We have assemblies with that prefix in sdk and other repos as well.

Reply:
Are you sure? I think those in SDK are all Microsoft.NET

Reply:
yes. here's a current list:
```
Microsoft.DotNet.ApiCompat.Task.dll
Microsoft.DotNet.ApiCompat.Task.resources.dll
Microsoft.DotNet.ApiCompatibility.dll
Microsoft.DotNet.ApiCompatibility.resources.dll
Microsoft.DotNet.ApiSymbolExtensions.dll
Microsoft.DotNet.ApiSymbolExtensions.resources.dll
Microsoft.DotNet.Cli.Utils.dll
Microsoft.DotNet.Cli.Utils.resources.dll
Microsoft.DotNet.Configurer.dll
Microsoft.DotNet.Configurer.resources.dll
Microsoft.DotNet.InternalAbstractions.dll
Microsoft.DotNet.NativeWrapper.dll
Microsoft.DotNet.PackageValidation.dll
Microsoft.DotNet.PackageValidation.resources.dll
Microsoft.DotNet.SdkResolver.dll
Microsoft.DotNet.TemplateLocator.dll
Microsoft.DotNet.TemplateLocator.resources.dll
```